awx/main/models/jobs.py

```diff
@@ -965,8 +1017,12 @@ class Meta:
             blank=True,
         )
     )
-    # Credentials needed for non-unified job / unified JT models
+    # Fields needed for non-unified job / unified JT models, because they are defined on unified models
     credentials = models.ManyToManyField('Credential', related_name='%(class)ss')
+    labels = models.ManyToManyField('Label', related_name='%(class)s_labels')
+    execution_environment = models.ForeignKey(
+        'ExecutionEnvironment', null=True, blank=True, default=None, on_delete=polymorphic.SET_NULL, related_name='%(class)s_as_prompt'
```

Comment:
As this is a `ForeignKey`, not a many-related field, it would be really nice to have help_text for OPTIONS, especially to echo the adherence to its `ask_` rule.

Reply:
This field is on the JobLaunchConfig the JobTemplate endpoint gets the POST options from ExecutionEnvironmentMixin. Is there somewhere in the API that you can see job launch configs?

Reply:
Schedules and WFJT nodes _are_ `JobLaunchConfig`s. It is an abstract model, but that just means that it has content shared by other concrete models. Those I list are not the _only_ concrete model that subclasses this and inherits its fields, but they are the easiest examples because they are listed and editable in the API the same way other normal resources are.

Reply:
Good call on the inheritance from schedules. The schedules endpoint was not display help text. That has been resolved.